superbench/executor/executor.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""SuperBench Executor."""
+
+from pathlib import Path
+
+from omegaconf import ListConfig
+
+from superbench.benchmarks import Platform, Framework, BenchmarkRegistry
+from superbench.common.utils import SuperBenchLogger, logger
+
+
+class SuperBenchExecutor():
+    """SuperBench executor class."""
+    def __init__(self, sb_config, docker_config, output_dir):
+        """Initilize.
+
+        Args:
+            sb_config (DictConfig): SuperBench config object.
+            docker_config (DictConfig): Docker config object.
+            output_dir (str): Dir for output.
+        """
+        self._sb_config = sb_config
+        self._docker_config = docker_config
+        self._output_dir = output_dir
+
+        self.__set_logger('sb-exec.log')
+        logger.info('Executor uses config: %s.', self._sb_config)
+        logger.info('Executor writes to: %s.', self._output_dir)
+
+        self.__validate_sb_config()
```

Comment:
`self.__validate_sb_config()` should return true if it pass the validation. If not, what should we do?

Reply:
updated comment